src/scipp/logging.py

```diff
@@ -3,6 +3,8 @@
 # @author Jan-Lukas Wynen
 """
 Utilities for managing scipp's logger and log widget.
+
+See the `Logging <../../reference/logging.rst>`_ reference for an overview.
```

Comment:
Maybe use absolute link, so if someone uses `help(scipp.logging)` in a notebook the link works?